pkgs/development/tools/misc/coreboot-toolchain/default.nix

```diff
@@ -0,0 +1,99 @@
+{ lib, stdenvNoCC, fetchurl, fetchgit,
+  gnumake, patch, zlib, git, bison,
+  flex, gnat11, curl, perl
+}:
+
+let
+  version_coreboot	= "4.14";
+
+  version_gmp		= "6.2.0";
+  version_mpfr		= "4.1.0";
+  version_mpc		= "1.2.0";
+  version_gcc		= "8.3.0";
+  version_binutils	= "2.35.1";
+  version_acpica	= "20200925";
+  version_nasm		= "2.15.05";
+
+  tar_name_gmp = "gmp-${version_gmp}.tar.xz";
+  tar_gmp = fetchurl {
+    url = "https://ftpmirror.gnu.org/gmp/${tar_name_gmp}";
+    sha256 = "09hmg8k63mbfrx1x3yy6y1yzbbq85kw5avbibhcgrg9z3ganr3i5";
+  };
```

Comment:
Why are we not using the sources from nixpkgs?

Reply:
I'm not sure if I understand your question correctly. Using the NixOS packages is not possible since the coreboot toolchain requires specific versions and also some patches in addition. Also, the components might be built with different compiler flags. Porting all that stuff to NixOS requires too much effort. So it's just easier to use the buildgcc script.